habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+
+# TODO: is there a better way to do this more generaly? Maybe pybind a cmake compiled DATA_DIR?
+dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
You are doing a lot of os.path.joins. You may want to look into PathLib (now standard as of Python 3.6)